### PR TITLE
Update Unifont from 5.1 to 13.0.04

### PR DESCRIPTION
**Pull request by patrikjuvonen**
## Summary
- Font contains various additions, changes and fixes
- Adds a new build action `install_unifont.lua` which is run via `win-create-projects.bat`
- Uses our [unifont repository](https://github.com/multitheftauto/unifont)
- Allows client scripts to use unifont via drawing functions
- Follow up PR for #1875
- Changelogs
  - http://unifoundry.com/unifont/index.html

### CODE REVIEW

**Comment by sbx320** [comment]
Wouldn't it be easier to add this to `install_data` rather than to add another buildaction?

**Comment by patrikjuvonen** [comment]
> Wouldn't it be easier to add this to `install_data` rather than to add another buildaction?

I thought about it, but I'm sort of conflicted. Currently install_data mostly affects netc pretty much so adding Unifont stands out as an odd one, but I don't know. I could move it.

**Review by Dutchman101 — APPROVED** [review]
Making it available to scripts is pretty nice as well!

File: utils/buildactions/install_unifont.lua [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+			return
+		end
+
+		-- Download Unifont
+		print("Downloading Unifont...")
+		if not http.download_print_errors(UNIFONT_BASEURL..UNIFONT_TAG.."/"..UNIFONT_FILENAME, archive_path) then
+			return
+		end
+
+		print("Unifont updated.")
```
Comment (botder) [inline utils/buildactions/install_unifont.lua:40]: Missing SHA256 hash verification for the downloaded unifont file here

**Review by botder — CHANGES_REQUESTED** [review]

File: utils/buildactions/install_unifont.lua (hunk 1 above)
Comment (patrikjuvonen) [reply utils/buildactions/install_unifont.lua:40]: Thanks, added.

**Review by botder — APPROVED** [review]
I have tested the font with a regular GUI label (height: 15) and here are the results:
Font size 9 and aliasing-enabled are the defaults for unifont.

grafik

**Pull request merged**